### PR TITLE
mech-lore: Change exp check to Engineering

### PR DESCRIPTION
**Pull request by simiydr**
Update the mech-lore script to use Engineering instead of Mechanical Lore.

### CODE REVIEW

File: mech-lore.lic [hunk 1]
```diff
@@ -42,7 +42,7 @@ class MechLore
       case bput("braid my #{item}", 'You need to have more', 'Roundtime', 'You need both hands to do that', 'You can\'t braid the .* into your braided', 'is already as long as you can make it', 'You are in no condition')
       when 'Roundtime'
         rt = reget(10, 'Roundtime').last.scan(/\d+/).first.to_i
-        break if rt >= 8 || DRSkill.getxp('Mechanical Lore') >= 30
+        break if rt >= 8 || DRSkill.getxp('Engineering') >= 30
```
Comment (Sarvatt) [inline mech-lore.lic]: I wish we could rename this to braid.lic but yeah I've mad the same change locally and forgot to push it. It needs changing.

Comment (simiydr) [reply mech-lore.lic]: We could rename this braid.lic and then just put a stub of a mech-lore.lic script that just starts braid.lic instead.

Comment (Sarvatt) [reply mech-lore.lic]: that's what I recommended to do to switch to town-training.lic instead of crossing-training.lic but it got pushback :D

Comment (Sarvatt) [reply mech-lore.lic]: that would be a big +1 from me though.

Comment (Sarvatt) [reply mech-lore.lic]: this script is essentially useless now, doesn't teach past like 50 engineering

Comment (simiydr) [reply mech-lore.lic]: I'm using it on a brand new toon and will track how it goes for me. Early ranks are going like mech lore used to.

**Pull request merged**